src/platforms/hosted/serial_win.c

```diff
@@ -256,8 +347,15 @@ bool serial_open(const bmda_cli_options_s *const cl_opts, const char *const seri
 
 void serial_close(void)
 {
-	CloseHandle(port_handle);
-	port_handle = INVALID_HANDLE_VALUE;
+	if (port_handle != INVALID_HANDLE_VALUE) {
+		CloseHandle(port_handle);
+		port_handle = INVALID_HANDLE_VALUE;
+		WSACleanup();
```

Comment:
Does this call not need to happen in the network `if` rather than this one?

Reply:
Oof, that's a really good catch.